antsibull/data/sphinx_init/conf_py.j2

```diff
@@ -0,0 +1,31 @@
+# This file only contains a selection of the most common options. For a full list see the
+# documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+project = 'Ansible collections'
+copyright = 'Ansible contributors'
+
+html_short_title = 'Ansible Collections Documentation'
+
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx', 'sphinx_antsibull_ext']
```

Comment:
Why not use the new Ansible theme?

Reply:
I'll switch once there's a release with https://github.com/ansible-community/sphinx_ansible_theme/pull/38.

Reply:
@felixfontein I've just released v0.8.0

Reply:
Implemented in efb19e3a1f56dfc724fc6fe9f57bdb01c7edb541.